src/raftstore/store/snap.rs

```diff
@@ -874,12 +872,10 @@ impl Snapshot for Snap {
         debug!("deleting {}", self.path());
         for cf_file in &self.cf_files {
             delete_file_if_exist(&cf_file.tmp_path);
-            if file_exists(&cf_file.path) {
-                let mut size_track = self.size_track.wl();
-                *size_track = size_track.saturating_sub(cf_file.size);
-            }
-            delete_file_if_exist(&cf_file.path);
             delete_file_if_exist(&cf_file.clone_path);
+            if delete_file_if_exist(&cf_file.path) {
+                self.size_track.fetch_sub(cf_file.size, Ordering::SeqCst);
```

Comment:
If overflow, can we not remove the snapshot anymore?

I still worry about it.

Reply:
There won't be any overflow. We increase `size_track` when cf files is renamed from tmp files, and decrease it after delete disk files success.